packages/cbioportal-ts-api-client/src/generated/CBioPortalAPIInternal.ts

```diff
@@ -196,6 +190,30 @@ export type ClinicalDataFilter = {
 
         'values': Array < DataFilterValue >
 
+};
+export type ClinicalDataPage = {
```

Comment:
seems odd to me that the API would have a type for ClinicalDataPage.   I would think at the API level that it should handle pagination in the same way as it does for GET requests.  Though one problem I see with the GET pagination is that it doesn't tell you in the response how many total items there are.   Let's discuss.